helm-configs/loki/README.md

```diff
@@ -0,0 +1,27 @@
+# Setting up Loki
+
+## Add the grafana helm repo
+```bash
+helm repo add grafana https://grafana.github.io/helm-charts
+helm repo update
+```
+
+## Install the helm chart
+
+You will need to make changes depending on how you want to configure loki. Example files are included in this directory choose one relevant to your deploy
+
+```bash
+helm upgrade --install --values my-loki-helm-overrides.yaml loki grafana/loki --create-namespace --namespace grafana
```

Comment:
Small nit:

--values my-loki-helm-overrides.yaml

Can we change this to the real file name ? Assuming we are using swift, lets change that to swift file ?

Reply:
The way the helm chart is you actually store real credentials in the helm chart file.  There is an option of configtype of either ConfigMap or Secret and will either put the loki config in a ConfigMap or Secret depending on the value.  I guess we could put fake values here for now.  Would that work?